MatchingGame.js

```diff
@@ -2,12 +2,10 @@ const section = document.querySelector("section");
 const playerLivesCount = document.querySelector("span");
 let playerLives = 10;
 
-
 // Link text
 playerLivesCount.textContent = playerLives;
 
-
-//Generate the data
+// Generate the data
 const getData = () => [
```

Comment:
I see that we are duplicating contents of the array in order to get pairs of each card, which is nice but not very flexible, adds unnecessary lines of code, and is harder to maintain (have to edit each card in multiple places). Try redoing this data modeling by using an additional variable for the pairs like:
```
{ imgSrc: "images/Hashirama.jpeg", name: "Hashirama", count: 2}
```
This way you can make a game with triple guesses or something in the future easily, and less duplication. Alternatively you can add a constant somewhere like
```
let CARD_COUNT = 2
```
and remove the duplicates from this array. Both of these things will require changing the `randomize()` function